docs/reference/aggregations/metrics/top-metrics-aggregation.asciidoc

```diff
@@ -54,7 +52,7 @@ The `sort` field in the metric request functions exactly the same as the `sort`
 <<request-body-search-sort, search>> request except:
 * It can't be used on <<binary,binary>>, <<flattened,flattened>, <<ip,ip>>,
 <<keyword,keyword>>, or <<text,text>> fields.
-* It only supports a single sort value.
+* It only supports a single sort value so which document wins ties is not specified.
```

Comment:
Is the behavior around ties deterministic, i.e. if no new documents are added will the aggregation always return the same results in the presence of ties?  Or do ties essentially result in random results?  Is it possible/worth it to return a flag indicating a tie happened, or do we not even know it at aggregation time?

I have gotten burned hard by users who think millisecond resolution time stamps are unique identifiers...

Reply:
`BucketedSort` is deterministic, but Lucene is only sort of deterministic. If you don't write anything *and* it doesn't merge any segments then it'll return the same thing every time. But writes and merges are async from, well, everything.

A flag for ties would require an extra bit for every bucket, I think. We'd need to store if the bottom entry hit a tie. We could derive whether or not there are ties from there. But that doesn't seem worth it.